resources/projects/VB.NET-ECommerce/App_Data/PublishProfiles/cis466shoestore.pubxml

```diff
@@ -0,0 +1,43 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This is from https://github.com/fchau/VB.NET-ECommerce, right? There's no license in that repo. Is this safe to embed here? [No license generally means you can't copy it](https://choosealicense.com/no-permission/).

Reply:
Thanks for pointing it. My fault, i didn't look at the missing license